MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 include dash_table/bundle.js
 include dash_table/bundle.js.map
+include dash_table/async~*.js
+include dash_table/async~*.js.map
```

Comment:
Where does the `~` come from? I suppose it's OK, just looks kinda weird, and I know Windows uses `~` for some system-related things...

Reply:
It's default naming that comes from webpack when using chunking groups: https://github.com/plotly/dash-table/blob/exp-dynamic/.config/webpack/base.js#L97